infrastructure/cloud-formation/worker.yml

```diff
@@ -22,31 +24,6 @@ Resources:
       PathPart: worker
       ParentId: !Ref RestApiRootId
 
-  WorkerTable:
```

Comment:
Because this was living inside the worker.yml we had a circular logic situation. Now I know for future projects that all data stores should probably live either in their own yml file or be moved to the base level. It's fine to make this change now, but it would have been a big pain in the rear if we had had any data.